js/src/figure.js

```diff
@@ -142,7 +150,8 @@ var FigureView = widgets.DOMWidgetView.extend( {
         this.wire_box.add(make_line(-0.5, -0.5+1, -0.5, -0.5, -0.5+1, -0.5+1, this.axes_material))
         this.wire_box.add(make_line(-0.5+1, -0.5+1, -0.5, -0.5+1, -0.5+1, -0.5+1, this.axes_material))
 
-        this.camera.position.z = 2
+        // set a good intial z for any fov angle
+        this.camera.position.z = 2 * this.getTanDeg(45/2) / this.getTanDeg(this.model.get("camera_fov")/2)
```

Comment:
(Updated) Maybe add 45 as a constant `INITIAL_VIEW_ANGLE` (or more descriptive) to avoid magic numbers?